nominal/core/filetype.py

```diff
@@ -87,6 +115,16 @@ class FileTypes:
     MCAP: FileType = FileType(".mcap", "application/octet-stream")
     # https://issues.apache.org/jira/browse/PARQUET-1889
     PARQUET: FileType = FileType(".parquet", "application/vnd.apache.parquet")
+    PARQUET_GZ: FileType = FileType(".parquet.gz", "application/octet-stream")
+    PARQUET_TARBALL: FileType = FileType(".parquet.tar.gz", "application/x-tar")
```

Comment:
this is super well-ackchually but can we go with `PARQUET_TAR_GZ` since I think a tarball is a tar with _any_ compression